config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -278,5 +278,7 @@
          files="src[\\/]xdocs[\\/]checks[\\/]whitespace[\\/]nowhitespacebeforecasedefaultcolon.xml.template"/>
   <suppress id="propertiesMacroMustExist"
          files="src[\\/]xdocs[\\/]filters[\\/]suppresswarningsfilter.xml.template"/>
+  <suppress id="propertiesMacroMustExist"
+            files="src[\\/]xdocs[\\/]checks[\\/]coding[\\/]constructorsdeclarationgrouping.xml.template"/>
```

Comment:
Please put it to "coding" group of suppression

Reply:
done.